Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1824,6 +1824,7 @@ a_Player:OpenWindow(Window);
 				GetHealth = { Return = "number" },
 				AddEntityEffect = { Params = "EffectType, {{cEntityEffect}}", Return = "", Notes = "Applies an entity effect" },
 				RemoveEntityEffect = { Params = "EffectType", Return = "", Notes = "Removes a currently applied entity effect" },
+				HasEntityEffect = { Return = "true, if the entity effect is currently applied", Params = "EffectType", Notes = "Checks, if an entity effect is currently applied" }
```

Comment:
The `Return` value should only specify the type of the return value(s). The description should be in `Notes`.
This is mainly for the ZBS IDE which can read these descriptions and then provide code completion based on them.